### ISSUE TITLE
TODO: capture generated columns in all sqldrivers 

### ISSUE THREAD

**Issue by ssweber** [PySimpleSQL/pysimplesql#314]
We now add generated columns in Datasets, , don’t try to update or insert into generated columns, and make generated columns “readonly” in CellEdit.

- [x] sqlite
- [x] MySQL
- [x] Postgres
- [x] Sqlserver
- [x] Msaccess
- [ ] 

Just putting this here for future reminder.

**Comment by PySimpleSQL** [comment]
Thanks for this, it's actually something I use often in my own projects and have been trying to think of a good work around.  Doing them at the driver level makes a lot of sense!

**Comment by ssweber** [comment]
For sure! I found it super useful in the orders example

I didn’t see the need to distinguish between stored vs virtual generated, but I went ahead and copied it over as-is (eg hidden = 1 or 2 for SQLite), but it still works for generated, not generated.

I’ll probably do the same for the other drivers (MySQL stores it as “, AND EXTRA = 'VIRTUAL GENERATED' OR EXTRA = 'STORED GENERATED' ) unless you want to standardize on a constant convention.
